components/modal/confirm.jsx

```diff
@@ -5,8 +5,24 @@ import Icon from '../icon';
 import Button from '../button';
 import objectAssign from 'object-assign';
 
-export default function (config) {
-  const props = objectAssign({}, config || {});
+const defaultLocale = {
+  okText: '确定',
+  cancelText: '取消',
+  justOkText: '知道了',
+};
+
+let runtimeLocale = { ...defaultLocale };
+
+export function changeConfirmLocale(newLocale) {
+  if (newLocale) {
+    objectAssign(runtimeLocale, newLocale);
+  } else {
+    runtimeLocale = defaultLocale;
```

Comment:
fix in: https://github.com/ant-design/ant-design/commit/b8d040f9438c464b02f1c70b1e6ebf636eb3cab2